cylc/flow/network/schema.py

```diff
@@ -1896,6 +1923,8 @@ class Meta:
             algorithm by artificially satisfying outputs of tasks.
 
             By default this makes tasks appear as if they succeeded.
+
+            Valid for: running workflows.
```

Comment:
Technically valid for paused workflows, however, the changes won't take effect until the workflow is resumed. So there might be an argument for restricting this to just running workflows, but that could backfire by making it difficult to intervene with a workflow whilst paused.

```suggestion
            Valid for: paused, running workflows.
```